admin/tests-integration/sql.service.spec.js

```diff
@@ -68,8 +68,8 @@ describe('sql.service:integration', () => {
     expect(row.version).toBeUndefined()
   })
 
-  it('dates should be stored as UTC and preserve up to 3 milliseconds', async () => {
-    const fullDateFormat = '2017-07-16T14:01:02.123+01:00'
+  it('dates should be stored as UTC and preserve a less than 1000 millisecondsn difference', async () => {
```

Comment:
Perhaps this is the typo?

Reply:
I’m not sure 🤔 best run it through a spell checker

Reply:
😆